src/glfw3.jl

```diff
@@ -177,13 +177,14 @@ const JOYSTICK_LAST          = JOYSTICK_16
 # Error codes
 const NOT_INITIALIZED        = 0x00010001  # GLFW has not been initialized.
 const NO_CURRENT_CONTEXT     = 0x00010002  # No context is current for this thread.
-const INVALID_ENUM           = 0x00010003  # One of the enum parameters for the function was given an invalid enum.
-const INVALID_VALUE          = 0x00010004  # One of the parameters for the function was given an invalid value.
+const INVALID_ENUM           = 0x00010003  # One of the arguments to the function was an invalid enum value.
```

Comment:
I'd like to, but the only thing stopping me right now is that creating an enum blows up if the integer isn't known. Is there a way to fix that?